README.md

```diff
@@ -14,6 +14,7 @@ Deck is a kanban style organization tool aimed at personal planning and project
 - Discuss with your team using comments
 - Keep track of changes in the activity stream
 - Get your project organized
+- [Migrate](https://github.com/maxammann/trello-to-deck) from Trello
```

Comment:
I'm happy to add this to the readme, though we should have a dedicated section for 3rd party integrations like the mobile apps section as this list is the general feature list of the app itself.